main.go

```diff
@@ -37,7 +37,7 @@ func main() {
 
 	// payslips for each month
 	payslipFn := requestFnPayslip(cfg, cookie)
-	for cur := cfg.from.Time; cfg.to.After(cur); cur = cur.AddDate(0, 1, 0) {
+	for cur := cfg.from.Time; cfg.to.BeforeOrEqual(&cur); cur = cur.AddDate(0, 1, 0) {
```

Comment:
just `!cur.After(cfg.to)`, don't need the BeforeOrEqual method

Reply:
Oh yes, [De Morgan's Law](https://www.cuemath.com/data/de-morgans-law/), I just hate negations in logic, trying to be positive ;).
Sure can fix it...